modules/checklist/assets/js/components/checklist-item.js

```diff
@@ -0,0 +1,65 @@
+import Button from '@elementor/ui/Button';
+import Card from '@elementor/ui/Card';
+import CardContent from '@elementor/ui/CardContent';
+import CardActions from '@elementor/ui/CardActions';
+import Typography from '@elementor/ui/Typography';
+import ListItemButton from '@elementor/ui/ListItemButton';
+import ListItemIcon from '@elementor/ui/ListItemIcon';
+import ListItemText from '@elementor/ui/ListItemText';
+import Collapse from '@elementor/ui/Collapse';
+import ChevronDownIcon from '@elementor/icons/ChevronDownIcon';
+import RadioButtonUncheckedIcon from '@elementor/icons/RadioButtonUncheckedIcon';
+import CardMedia from '@elementor/ui/CardMedia';
+import Link from '@elementor/ui/Link';
+
+function CheckListItem( props ) {
+	const { id, title, description, link, CTA } = props.step,
+		[ expanded, setExpanded ] = React.useState( false );
+
+	const handleExpandClick = () => {
+		setExpanded( ! expanded );
+	};
```

Comment:
Please check with Doron if multiple listitems can be expanded at once, if not - then the handler should be where the map is, using exandedIndex state or something like that